attributes/default.rb

```diff
@@ -1,8 +1,48 @@
+#
+# Author:: Matt Ray <matt@chef.io>
+# Author:: Ovais Tariq <me@ovaistariq.net>
+# Cookbook Name:: drbd
+# Attributes:: default
+#
+# Copyright 2009, Chef Software, Inc.
+# Copyright 2015, Ovais Tariq
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The DRBD packages and their versions
+default["drbd"]["packages"] = Hash.new
```

Comment:
There's no need to do a Hash.new here. Just make the attribute equal the result of the case statement.